doc/source/_static/numpy.css

```diff
@@ -1,10 +1,11 @@
 @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap');
 
 .navbar-brand img {
-   height: 75px;
+  height: 75px;
 }
+
 .navbar-brand {
-   height: 75px;
+  height: 75px;
 }
```

Comment:
For another time, do you believe it would be worth having a pre-comit to format the css in the numpy repo ?

Reply:
I think so! I couldn't find many within the `pre-commit` framework that seem to be regularly maintained, but Node.js-based tools exist, such as https://github.com/stylelint/stylelint that one can use with `npm`, or better `npx`.